worker-d1/README.md

```diff
@@ -0,0 +1,59 @@
+# Template: worker-d1
+
+**Note: requires D1 Beta Access**
+
+This project is based off the Default Typescript Worker starter. To create a new project like this, run the following:
+
+```sh
+npx wrangler@d1 init -y
+```
+
+Then copy the `src` and the `data` directory from this template into your project.
+
+Alternatively, clone this repo directly:
+
+```
+npm init cloudflare northwind-demo worker-d1
+```
```

Comment:
```suggestion
```sh
npm init cloudflare northwind-demo worker-d1
```
```

Reply:
Is this the link to the repo?

Reply:
This will be how people can clone this folder once this PR is merged, as far as I understand it.